lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -448,6 +493,157 @@ def make_PUT_request(
         https_connection.close()
 
 
+def send_comp_and_pen_opt_in_confirmation(va_profile_id: int) -> Optional[HTTPResponse]:
+    """
+    Send an opt-in confirmation SMS notification based on user's VAProfile ID.
+
+    Args:
+        va_profile_id (int): The VA profile ID of the user the notification should be sent to.
+    """
+
+    try:
+        # Personalization for opt-in confirmation notification SMS based on cutoff date
+        # to enroll in monthly Comp and Pen notification
+        now = datetime.now(timezone.utc)
+        cutoff_datetime = datetime(now.year, now.month, *COMP_AND_PEN_OPT_IN_CUTOFF_TIME_UTC, tzinfo=timezone.utc)
+
+        if now < cutoff_datetime:
+            month_personalisation = calendar.month_name[now.month]
+        else:
+            next_month = (now.month % 12) + 1
+            month_personalisation = calendar.month_name[next_month]
+
+        sms_data = json.dumps(
+            {
+                'template_id': COMP_AND_PEN_OPT_IN_TEMPLATE_ID,
+                'recipient_identifier': {'id_type': 'VAPROFILEID', 'id_value': str(va_profile_id)},
+                'sms_sender_id': COMP_AND_PEN_SMS_SENDER_ID,
+                'personalisation': {'month-name': month_personalisation},
+            }
+        )
+
+        logger.debug('Sending Comp and Pen opt-in confirmation SMS notification vaProfileId %s', va_profile_id)
+
+        conn = HTTPSConnection('{}.api.notifications.va.gov'.format(NOTIFY_ENVIRONMENT), context=ssl_context)
```

Comment:
Why aren't we using the Requests library here?
Also, can we use f-strings instead of .format?

Reply:
I also did not think we wanted to load another library with a _similar functionality_ into this lambda.

Reply:
I meant replace the thing that exists with requests (future work)

Reply:
I think I was confused on how we are expected to use F string vs. Format. I have updated this string and moved the domain name to a constant.